go/build.sh

```diff
@@ -67,6 +67,9 @@ export COMMANDS=(
 export TERRAFORM_COMMANDS=(
 	vendor/github.com/hashicorp/terraform
 	vendor/github.com/hashicorp/terraform/builtin/bins/...
+)
+
+export TERRAFORM_CUSTOM_COMMANDS=(
```

Comment:
terraform's own github provider overrides koding's github provider. we disable terraform's github provider.  we will change koding github provider and enable back terraform one. But for a while we will use koding's github provider